src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -278,6 +278,12 @@ public class ProtoCompile : ToolTask
         /// </summary>
         public string[] OutputOptions { get; set; }
 
+        /// <summary>
+        /// Additional arguments that will be passed unmodified to protoc (and before any file names).
+        /// For example, "--experimental_allow_proto3_optional"
+        /// </summary>
+        public string[] AdditionalProtocArguments { get; set; }
```

Comment:
One concern is that this is a string array (one string per argument) rather than a single string (containing all arguments). I'm not sure if that'll make any practical difference in practice with how these get serialized to a "response" file that MSBuild creates. Regardless, I wanted to copy the style of other options that used arrays.

Reply:
I think the approach you used is the correct one.